test/test_prototype_models.py

```diff
@@ -117,13 +119,22 @@ def test_video_model(model_fn, dev):
     TM.test_video_model(model_fn, dev)
 
 
+@needs_cuda
+@pytest.mark.parametrize("model_builder", (models.optical_flow.raft_large, models.optical_flow.raft_small))
+@pytest.mark.parametrize("scripted", (False, True))
+@run_if_test_with_prototype
+def test_raft(model_builder, scripted):
+    TM.test_raft(model_builder, scripted)
+
+
 @pytest.mark.parametrize(
     "model_fn",
     TM.get_models_from_module(models)
     + TM.get_models_from_module(models.detection)
     + TM.get_models_from_module(models.quantization)
     + TM.get_models_from_module(models.segmentation)
-    + TM.get_models_from_module(models.video),
+    + TM.get_models_from_module(models.video)
+    + TM.get_models_from_module(models.optical_flow),
```

Comment:
Once weights are supported, I'll need to edit the code below a bit because the raft model needs 2 inputs instead of 1.

Reply:
Hmm... You are right.

If we can't modify the method yet to make it ready, then there is no point adding the `optical_flow` module yet. Up to you on whether you want to revert or update the method.

Reply:
I think there's value in keeping it: currently the test is skipped because there are no weights, but once I add the weights the test will fail. This is a good thing since it will prevent us from forgetting to add this test again?

Reply:
Sounds good to me.